desktop/core/src/desktop/js/apps/editor/execution/sqlExecutable.ts

```diff
@@ -99,6 +100,38 @@ export default class SqlExecutable extends Executable {
   }
 
   async internalExecute(): Promise<ExecuteApiResponse> {
+    if (this.parsedStatement && /CREATE|DROP/i.test(this.parsedStatement.firstToken)) {
+      let match = this.parsedStatement.statement.match(
+        /(?:CREATE|DROP)\s+TABLE\s+(?:IF\s+(?:NOT\s+)?EXISTS\s+)?(?:`([^`]+)`|([^;\s]+))\..*/i
```

Comment:
Put the expressions in const on top, i.e

```
const TABLE_DDL_REGEX = /.../i;
const DB_DDL_REGEX =
```

Reply:
Also, good to add views to it, i.e.

`\s+TABLE\s+   =>   \s+(?:TABLE|VIEW)\s+`